G/Gloo/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Gloo"
+version = v"0.0.20200317"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/facebookincubator/gloo.git", "113bde13035594cafdca247be953610b53026553"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/gloo
+mkdir build
+cd build
+cmake \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_SHARED_LIBS=ON \
+    -DUSE_LIBUV=ON \
+    ..
+cmake --build . -- -j $nproc
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+filter!(p -> nbits(p) == 64, platforms) # Gloo can only be built on 64-bit systems
+filter!(!Sys.iswindows, platforms) # Windows support will be available from 20200910, i.e. 881f7f0dcf06f7e49e134a45d3284860fb244fa9
+platforms = expand_cxxstring_abis(platforms)
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libgloo", :libgloo),
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency("LibUV_jll", v"2.0.0"),
```

Comment:
It doesn't seem to link to libuv:
```console
% readelf -d x86_64-linux-gnu-cxx11/lib/libgloo.so|head

Dynamic section at offset 0xd7e40 contains 26 entries:
  Tag        Type                         Name/Value
 0x0000000000000001 (NEEDED)             Shared library: [libpthread.so.0]
 0x0000000000000001 (NEEDED)             Shared library: [libstdc++.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libm.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libgcc_s.so.1]
 0x0000000000000001 (NEEDED)             Shared library: [libc.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [ld-linux-x86-64.so.2]
 0x000000000000000e (SONAME)             Library soname: [libgloo.so]
```
It's probably doing static linking?  I see lots of `uv_*` symbols in this library.
```suggestion
    BuildDependency("LibUV_jll", v"2.0.0"),
```

Reply:
Yeah, Julia's fork of libuv has inconsistent versioning

Reply:
Right, I remember reading something about the Julia being significant patches ahead/beside of main libuv.